src/test/resources/jira_db_schema/V1__create_tables.sql

```diff
@@ -0,0 +1,98 @@
+create table if not exists project
```

Comment:
Can you add a comment to the top of the file explaining where this comes from? I.e. a schema dump of which version of Jira, taken on what date.